back/templates/custom/00_struct.go.tpl

```diff
@@ -0,0 +1,69 @@
+{{- $alias := .Aliases.Table .Table.Name -}}
+{{- $orig_tbl_name := .Table.Name -}}
+
+// {{$alias.DownSingular}}View is the view displayed to clients.
+type {{$alias.DownSingular}}View struct {
+	{{- range $index, $column := .Table.Columns -}}
+	{{- $colAlias := $alias.Column $column.Name -}}
+	{{- $orig_col_name := $column.Name -}}
+
+	{{if ignore $orig_tbl_name $orig_col_name $.TagIgnore -}}
+		{{if eq (index $column.Type 0) '*'}}
+			{{$colAlias}} {{$column.Type}} `{{generateIgnoreTags $.Tags}}boil:"{{$column.Name}}" json:"-" toml:"-" yaml:"-"`
+		{{- else -}}
+			{{$colAlias}} *{{$column.Type}} `{{generateIgnoreTags $.Tags}}boil:"{{$column.Name}}" json:"-" toml:"-" yaml:"-"`
+		{{- end -}}
+	{{ else -}}
+
+	{{- /* render column alias and column type */ -}}
+	{{if eq (index $column.Type 0) '*'}}
+		{{ $colAlias }} {{ $column.Type -}}
+	{{- else -}}
+		{{ $colAlias }} *{{ $column.Type -}}
+	{{- end -}}
+
+	{{- /*
+	  handle struct tags
+	  StructTagCasing will be replaced with $.StructTagCases
+	  however we need to keep this backward compatible
+	  $.StructTagCasing will only be used when it's set to "alias"
+    */ -}}
```

Comment:
Why comment ? deadcode or for futur pr ?

Reply:
It's taken from SQLBoiler's template (as we are deriving from it), I can remove it if necessary

Reply:
Not fan of having commented code. Bad practice in my opinion.

At least must have a clear description that is from the SQLBoiler's template